README.md

```diff
@@ -54,6 +54,10 @@ GitLab LDAP configuration; if `gitlab_ldap_enabled` is `true`, the rest of the c
 
 Gitlab timezone.
 
+    gitlab_validate_certs: yes
+
+Certificates verification. If no, SSL certificates will not be validated.
```

Comment:
This would read (after changing var name above): "Controls whether to validate certificates when downloading the GitLab installation repository install script." (and might be good to add some justification as to why someone would ever want this to _not_ be `yes`...